dashboards-observability/public/components/common/search/autocomplete.tsx

```diff
@@ -67,7 +67,7 @@ const statsCommands = [
 // Function to create the array of objects to be suggested
 const fillSuggestions = (str: string, word: string, items: any) => {
   const filteredList = items.filter(
-    (item: { label: string }) => item.label.startsWith(word) && word !== item.label
+    (item: { label: string }) => item.label.toLowerCase().startsWith(word.toLowerCase()) && word.toLowerCase().localeCompare(item.label.toLowerCase())
```

Comment:
why is both `startsWith` and `localeCompare` needed? Also maybe assign a variable to `word.toLowerCase()` so it's not calculated every loop

Reply:
There were some instances where !== was returning true for words that were the same, this seemed to fix that issue. An example was "Dest" and "dest". I'll make the variable though!

Reply:
i meant if you are comparing for equality, then `startsWith` is always true, is it necessary to check both?

Reply:
We check for both because we want to keep it if the label starts with the word but not keep it if they are the same. 
So if the user types "Dest" we want to suggest "DestCityName", "DestCountry" but not "Dest"

Reply:
oh you are using `!!localeCompare(..) === false` to check if it's equal, i feel it's a bit confusing but ok

Reply:
Oh yea, localeCompare returns 0 if they are equal which is definitely confusing. Should I add a comment there?